ldap/servers/plugins/replication/cl5_api.c

```diff
@@ -556,9 +560,6 @@ cl5ImportLDIF(const char *clDir, const char *ldifFile, Replica *replica)
         goto done;
     }
 
-    /* Set changelog state to import */
-    pthread_mutex_lock(&(cldb->stLock));
-
```

Comment:
IMHO the if (cldb->dbState == CL5_STATE_IMPORT) statement should be moved before testing that the state is not state is CL5_STATE_OPEN.

Reply:
> IMHO the if (cldb->dbState == CL5_STATE_IMPORT) statement should be moved before testing that the state is not state is CL5_STATE_OPEN.

Fixed